### PR TITLE
add 'title' parameter to 'alluvial()'

### PR DESCRIPTION
**Pull request by corybrunson**
Add a `title` parameter to `alluvial()` that is passed to the `main` parameter in the `plot()` call, plus a parameter description in the documentation. Then `devtools::check()`. This is in response to #35

### CODE REVIEW

File: R/alluvial.R [hunk 1]
```diff
@@ -45,7 +46,8 @@ alluvial <- function( ..., freq,
                      cex=par("cex"),
                      xlim_offset= c(0, 0),
                      ylim_offset= c(0, 0),
-                     cex.axis=par("cex.axis"))
+                     cex.axis=par("cex.axis"),
+                     title = "")
```
Comment (mbojan) [inline R/alluvial.R]: I think it would be better to have a `title=NULL` as default. Same as `plot`.

Comment (corybrunson) [inline R/alluvial.R]: (Sorry for the double-update. I forgot that pushing new commits automatically updates the pull request.)

Comment (mbojan) [inline R/alluvial.R]: That's fine, its actually a feature.

**Comment by mbojan** [comment]
Thanks!

File: R/alluvial.R (hunk 1 above)
Comment (corybrunson) [inline R/alluvial.R]: Oh, neat. If i send a PR from a branch, and then you accept it, will it result in a new branch in your own repo, which you then have the option of merging?

Comment (mbojan) [inline R/alluvial.R]: Yes, basically that's how it works. The advantage is that you can focus on working on a PR and push commits just like we just did while not mess with `master` which can be updated by smbdy else simultaneously.....

**Pull request merged**